spec/slack/web/api/errors/slack_error_spec.rb

```diff
@@ -4,13 +4,22 @@
 RSpec.describe Slack::Web::Api::Errors::SlackError do
   let(:client) { Slack::Web::Client.new }
 
-  it 'provides access to the response object', vcr: { cassette_name: 'web/auth_test_error' } do
```

Comment:
I would also leave that old test in-place and add checks for its `.message`, `.error`, `.response_metadata`, it's a simpler case, but will prevent breaking it in the future.